### ISSUE TITLE
[BUG] Hestia-PHP error icon after update to 1.3 

### ISSUE THREAD

**Issue by theChaosCoder** [hestiacp/hestiacp#1318]
There is a warning icon after my server made an auto update to v1.3. What does this mean? I don't see anything in the logs and I don't seem to have an error anywhere. No information is shown if I hover over the icon.

My webiste with php fpm 7.4 is working.

![image](https://user-images.githubusercontent.com/3579830/97404975-3e87ad80-18f7-11eb-99ae-bbea728099d1.png)
Where should I look to

### Operating system:
Debian 10 (latest updates)

### Hestia Control Panel version:
1.3

**Comment by ghost** [comment]
Run `apt update && apt install --reinstall hestia-php` to ensure the latest version is installed, from what we've gathered so far we believe this may be a bug in the UI itself and the latest update did in fact install correctly.

**Comment by theChaosCoder** [comment]
reinstall did not help, icon is still shown.

**Comment by theChaosCoder** [comment]
This should be the check for the ui part https://github.com/hestiacp/hestiacp/blob/c5c4d411cdd47caa683169d99280d7e223acbe27/web/templates/admin/list_updates.html#L75

https://github.com/hestiacp/hestiacp/blob/c5c4d411cdd47caa683169d99280d7e223acbe27/web/list/updates/index.php#L15
```
# v-list-sys-hestia-updates

PKG           VER     ARCH   UPDT  DATE
---           ---     ----   ----  ----
hestia        1.3.0   amd64  yes   2020-10-28
hestia-php    7.4.11  amd64  no    2020-10-28
hestia-nginx  1.19.3  amd64  yes   2020-10-28
```
-----------

And some command from here https://github.com/hestiacp/hestiacp/blob/0fe2480cd461792df79abd63b6cf0c02f63f7939/bin/v-list-sys-hestia-updates#L83

**apt-cache policy hestia-php**
```
hestia-php:
  Installed: 7.4.11-1
  Candidate: 7.4.11-1
  Version table:
 *** 7.4.11-1 500
        500 https://apt.hestiacp.com buster/main amd64 Packages
        100 /var/lib/dpkg/status
     7.4.11 500
        500 https://apt.hestiacp.com buster/main amd64 Packages
     7.4.10 500
        500 https://apt.hestiacp.com buster/main amd64 Packages
     7.4.6 500
        500 https://apt.hestiacp.com buster/main amd64 Packages
     7.3.11 500
        500 https://apt.hestiacp.com buster/main amd64 Packages
     7.3.4 500
        500 https://apt.hestiacp.com buster/main amd64 Packages 
```
**# apt-cache policy hestia-php | grep Candidate | cut -d ':' -f 2 | xargs**
`7.4.11-1`

**# dpkg-query -s hestia-php**
```
Package: hestia-php
Status: install ok installed
Priority: optional
Section: admin
Maintainer: HestaCP <info@hestiacp.com>
Architecture: amd64
Source: hestia-php
Version: 7.4.11-1
Depends: hestia, libzip5 | libzip4, unzip, libonig5 | libonig4 | libonig2
Description: hestia php-fpm
 hestia is an open source hosting control panel.
 hestia has a clean and focused interface without the clutter.
 hestia has the latest of very innovative technologies.
 hestia is a fork from VestaCP, special thanks to vestacp.com and Serghey Rodin
Homepage: https://www.hestiacp.com
```

**Comment by theChaosCoder** [comment]
Could this be the problem?
```
dpkg_data=$(dpkg-query -s hestia-php)
echo "$dpkg_data"|grep ^Version |cut -f 2 -d ' '|cut -f 1 -d \-
7.4.11
```

**7.4.11** vs **7.4.11-1** ?

**Comment by ScIT-Raphael** [comment]
yes, this is the problem, we'll work on a fix.

**Comment by jaapmarcus** [comment]
Yes, we will fix the issue with next service release. 

We had to release an emergency fix for Hestia-php due to issues during install

**Comment by theChaosCoder** [comment]
ok, I'm glad it's only a small ui bug :-)

**Comment by ghost** [comment]
Can you please run `wget -O $HESTIA/bin/v-list-sys-hestia-updates https://raw.githubusercontent.com/hestiacp/hestiacp/fix/1318-ui-update-versioning/bin/v-list-sys-hestia-updates && chmod +x $HESTIA/bin/v-list-sys-hestia-updates` and see if this resolves the issue for you?

**Comment by theChaosCoder** [comment]
Looks good

**Comment by julianarmando** [comment]
Is not only an icon problem, the default backend template of php-fpm stop working, all the sites using the default template are down, and, I move it to PHP-7_4 template and working again.

**Comment by ScIT-Raphael** [comment]
@julianarmando have you used webalizer statistics? Does it work again, when you now switch back to default?

**Comment by julianarmando** [comment]
> @julianarmando have you used webalizer statistics? Does it work again, when you now switch back to default?

No, I have the web statistics in "none" for all my sites.

When the error appears, I see in the error log this...

`No protocol handler was valid for the URL /index.php (scheme '|fcgi'). If you are using a DSO version of mod_proxy, make sure the proxy submodules are included in the configuration using LoadModule.`

**Comment by theChaosCoder** [comment]
oh I only use the PHP-7_4 template so I did not notice it.

**Comment by jaapmarcus** [comment]
@julianarmando Please create a new issue to keep it separate. This issue is not related to your issue.

**Comment by theChaosCoder** [comment]
@julianarmando I tried the default template and it works too, so you seem have indeed a different issue.

**Comment by ScIT-Raphael** [comment]
The problem is already fixed and will be released with hestia 1.3.1.
